cinema/urls.py

```diff
@@ -1,10 +1,24 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
+from cinema.views import (
+    GenreDetail,
+    GenreList,
+    ActorDetail,
+    ActorList,
+    MovieViewSet,
+    CinemaHallViewSet
+)
 
-from cinema.views import movie_list, movie_detail
+router = routers.DefaultRouter()
+router.register("cinema_halls", CinemaHallViewSet, basename="cinema_hall")
+router.register("movies", MovieViewSet, basename="movie")
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("", include(router.urls)),
```

Comment:
No need to include if basepath is not provided. Instead use:
urlpatterns = [list of urls] + router.urls